sds_data_manager/stacks/ialirt_database_stack.py

```diff
@@ -0,0 +1,54 @@
+"""Stack for creating the IALIRT database."""
+
+from aws_cdk import (
+    RemovalPolicy,
+    Stack,
+)
+from aws_cdk import (
+    aws_dynamodb as ddb,
+)
+from constructs import Construct
+
+
+class IAlirtDatabaseStack(Stack):
+    """Stack for creating the IALIRT database."""
+
+    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+        """Initialize the IALIRT database stack.
+
+        Parameters
+        ----------
+        scope : Construct
+            The App object in which to create this Stack
+        construct_id : str
+            The ID (name) of the stack
+        **kwargs
+            Additional keyword arguments.
+
+        Note:
+        DynamoDB's table defines the key schema and any secondary indexes.
+        The non-key attributes are included items are added to the table.
+        """
+        super().__init__(scope, construct_id, **kwargs)
+
+        self.packet_data_table = ddb.Table(
+            self,
+            "PacketDataTable",
+            table_name="imap-packetdata-table",
+            # Change to RemovalPolicy.RETAIN to keep the table after stack deletion.
+            # TODO: change to RETAIN in production.
+            removal_policy=RemovalPolicy.DESTROY,
+            # Restore data to any point in time within the last 35 days.
+            # TODO: change to True in production.
+            point_in_time_recovery=False,
+            # Partition key (PK) = spacecraft time ugps.
+            partition_key=ddb.Attribute(
+                name="sct_vtcw_reset#sct_vtcw",
+                type=ddb.AttributeType.STRING,
+            ),
+            # Enable DynamoDB streams for real-time processing
+            stream=ddb.StreamViewType.NEW_IMAGE,
```

Comment:
Can this be added on/updated later? We aren't using the stream yet per @bryan-harter's comment.

Reply:
It looks like I have options for using streams, so I'll save this for another PR.